universal-login-commons/lib/core/utils/getGasPriceFromGasModes.ts

```diff
@@ -0,0 +1,7 @@
+import {GasMode, GasOption} from '../models/gasMode';
+
+export const getGasPriceFromGasModes = (gasModes: GasMode[], gasModeName: string, gasTokenAddress: string) => {
+  const mode = gasModes!.filter((gasPriceMode: GasMode) => gasPriceMode.name === gasModeName)[0];
+  const gasOption = mode && mode.gasOptions.filter((gasOption: GasOption) => gasOption.token.address === gasTokenAddress)[0];
```

Comment:
`mode && mode.` -> `mode!.` should throw on invalid input not silently fail

Reply:
✔️